src/helpers/requesters/ResourceRequester.js

```diff
@@ -0,0 +1,16 @@
+import BaseRequester from './BaseRequester';
+import { APIRoutes } from '../routes/routes';
+
+class ResourceRequester {
+	static async resources() {
+    const endpoint = APIRoutes.resourcePath();
+    try {
+      let response_json = await BaseRequester.get(endpoint);
+      return Promise.resolve(response_json);
+    } catch (error) {
+      return Promise.reject(error);
+    }
+  }
+}
+
+export default ResourceRequester;
```

Comment:
I don’t think you need this file since you can just use baserequester.get directly to achieve the same thing. I can refactor this for you in a later pr if you want